test/integration/simple_example/inspec.yml

```
@@ -0,0 +1,36 @@
+name: simple_example
+depends:
+  - name: inspec-gcp
+    git: https://github.com/inspec/inspec-gcp.git
+    version: ~> 0.10.0
```

The version attribute of git dependencies is currently broken.

```suggestion
    tag: v0.10.0
```